lib/oli_web/templates/layout/_account_header.html.eex

```diff
@@ -20,7 +20,7 @@
 
     <h3><%= Map.get(@conn.assigns, :title, "") %></h3>
     <div class="page-controls">
-      <%= link "Sign out", to: Routes.authoring_pow_session_path(@conn, :delete), method: :delete, class: "btn btn-sm btn-primary" %>
```

Comment:
We got hit on this for the security audit, the logout link should use the `delete` method vs a `get` method.